src/packageManager/downloadAndInstallPackages.ts

```diff
@@ -61,5 +62,7 @@ export async function downloadAndInstallPackages(packages: AbsolutePathPackage[]
                 catch (error) { }
```

Comment:
Should we do something about this?

Reply:
I dont think its that important. It just catches an error if the deletion of installFile throws an exception